### DISCUSSION TITLE
响应时间过长

### DISCUSSION THREAD

**Discussion by lovese** [labring/sealos#4573] [General]
sealos私有云部署完后，导入了证书，访问每个应用都是提示响应时间过长，这个怎么解决
![image](https://github.com/labring/sealos/assets/13268438/257b05cb-0e17-46d4-b0ee-58d491cb31e7)

**Comment by lovese** [comment]
![image](https://github.com/labring/sealos/assets/13268438/56661266-77c6-45f6-bbd6-9a5e2f5f8260)

**Comment by bxy4543** [comment]
感觉像是证书信任的问题， 或者重新打开浏览器试试
